docs/ChannelValue.md

```diff
@@ -48,21 +48,17 @@ You can override the classes used by Brightlayer UI by passing a `classes` prop.
 | root   | Styles applied to the root element                        |
 | icon   | Styles applied to the icon element                        |
 | text   | Styles applied to the text element                        |
-| prefix | Styles applied to the units element when used as a prefix |
```

Comment:
Was there a reason for this change?  Major breaking to remove classes.

Reply:
If you check the diff with dev/master you'll see there never was a prefix/suffix class. I added it in a previous commit and refactored to remove them.

Reply:
Makes sense! Thanks for clarification.